egs/tedlium/s5_r2/local/nnet3/tuning/run_lstm_1a.sh

```diff
@@ -101,22 +97,32 @@ for f in $train_data_dir/feats.scp $train_ivector_dir/ivector_online.scp \
 done
 
 if [ $stage -le 12 ]; then
-  echo "$0: creating neural net configs"
-  config_extra_opts=()
-  [ ! -z "$lstm_delay" ] && config_extra_opts+=(--lstm-delay "$lstm_delay")
-  steps/nnet3/lstm/make_configs.py  "${config_extra_opts[@]}" \
-    --feat-dir $train_data_dir \
-    --ivector-dir $train_ivector_dir \
-    --ali-dir $ali_dir \
-    --num-lstm-layers $num_lstm_layers \
-    --splice-indexes "$splice_indexes " \
-    --cell-dim $cell_dim \
-    --hidden-dim $hidden_dim \
-    --recurrent-projection-dim $recurrent_projection_dim \
-    --non-recurrent-projection-dim $non_recurrent_projection_dim \
-    --label-delay $label_delay \
-    --self-repair-scale-nonlinearity 0.00001 \
-  $dir/configs || exit 1;
+  echo "$0: creating neural net configs using the xconfig parser";
+
+  num_targets=$(tree-info $gmm_dir/tree | grep num-pdfs | awk '{print $2}')
+  [ -z $num_targets ] && { echo "$0: error getting num-targets"; exit 1; }
+
+  lstm_opts="decay-time=20 cell-dim=$cell_dim"
+  lstm_opts+=" recurrent-projection-dim=$recurrent_projection_dim"
+  lstm_opts+=" non-recurrent-projection-dim=$non_recurrent_projection_dim"
+
+  mkdir -p $dir/configs
+  cat <<EOF > $dir/configs/network.xconfig
+  input dim=100 name=ivector
+  input dim=40 name=input
+
+  # please note that it is important to have input layer with the name=input
+  # as the layer immediately preceding the fixed-affine-layer to enable
+  # the use of short notation for the descriptor
```

Comment:
I asked about label delay because I guess I had seen some warning/error before in another setup regarding label delay. Do you want me to run this command only to at least make sure it runs w/o problems?